extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/i18n/hue_de.properties

```diff
@@ -59,6 +79,40 @@ channel-type.hue.alert.state.option.LSELECT = Mehrfaches Blinken
 channel-type.hue.effect.label = Farbeffekt
 channel-type.hue.effect.description = Erm�glicht einen automatischen Farbwechsels.
 
+channel-type.hue.last_updated.label = Letzte Aktualisierung
+channel-type.hue.last_updated.description = Zeit, zu der sich dieser Wert ge�ndert hat
+channel-type.hue.lightlevel.label = Lichtst�rke
+channel-type.hue.lightlevel.description = Die aktuelle Lichtst�rke in Lux
+channel-type.hue.dark.label = Dunkel
+channel-type.hue.dark.description = Die Lichtst�rke ist unter dem Dunkel-Schwellwert
+channel-type.hue.daylight.label = Hell
+channel-type.hue.daylight.description = Die Lichtst�rke ist �ber dem Hell-Schwellwert
+channel-type.hue.dimmer_switch.label = Dimmer-Schalter
+channel-type.hue.dimmer_switch.description = Der zuletzt gedr�ckte Taster an dem Dimmer-Schalter
+channel-type.hue.temperature.label = Temperatur
+channel-type.hue.temperature.description = Die Temperatur
+
+channel-type.hue.dimmer_switch.options.option.1000 = An (Initial)
```

Comment:
The translations for the state options does not work. The key is wrong. Sry for my misleading example.
```
channel-type.hue.dimmer_switch.state.option.1000 = ...
```